Asset Uploader UI Component/Code/Asset Uploader/Objective-C/AssetUploader/ViewController.m

```diff
@@ -261,7 +278,44 @@ - (void)syncFinished
     [[AdobeLibraryManager sharedInstance] deregisterDelegate:self];
 }
 
-#pragma mark - Priavte methods
+#pragma mark - Notification handlers
+
+- (void)authDidLogout:(NSNotification *)notification
+{
+    if (self.rootLibDir.length > 0)
+    {
+        [AdobeLibraryManager removeLocalLibraryFilesInRootFolder:self.rootLibDir withError:nil];
+        
+        // Clean up the root folder.
+        NSFileManager *fm = [NSFileManager defaultManager];
+        [fm removeItemAtPath:[[self.rootLibDir stringByDeletingLastPathComponent] stringByDeletingLastPathComponent] error:nil];
+        self.rootLibDir = nil;
+        [[NSUserDefaults standardUserDefaults] removeObjectForKey:@"userSpecificPathKey"];
```

Comment:
Since the key is used in different methods, I think it’d make sense to create a static const or a `#define` to avoid any issues.